.eslintrc

```diff
@@ -0,0 +1,35 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "extends": [
+    "eslint:recommended",
+    "standard",
+    "prettier",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "plugins": ["@typescript-eslint", "prettier"],
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "ignorePatterns": ["dist", "node_modules", "examples", "scripts"],
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/ban-types": "off",
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/triple-slash-reference": "off",
+    "@typescript-eslint/no-this-alias": "off"
+  },
+  "overrides": [
+    {
+      "files": ["packages/crypto/**/*.js", "packages/docs/**/*.js"],
```

Comment:
Instead of globbing the whole crypto/docs js files, can we just glob the react apps instead? Because that way we can avoid js files in the non-react parts of the application

Reply:
Done, as discussed